modules/msk/genomenexus/vcf2maf/main.nf

```diff
@@ -0,0 +1,45 @@
+process GENOMENEXUS_VCF2MAF {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
```

Comment:
I think we should drop this `conda` line from the code.

I'm going to make an issue so we can document how to remove conda support. There seems to be some discrepancies between modules.